[cmake] remove usage of kodi-platform

olafhering opened the pull request:
The pvr.pctv binary addon does not use kodi-platform.
Remove existing references from CMakeLists.txt and debian/control.

Rechi commented on debian/control:
```diff
@@ -1,7 +1,7 @@
 Priority: extra
 Maintainer: hg1 <gerlach.holger@gmail.com>
-Build-Depends: debhelper (>= 9.0.0), cmake, libkodiplatform-dev (>= 16.0.0),
+Build-Depends: debhelper (>= 9.0.0), cmake, libp8-platform-dev
```
please fix the syntax error in this line

olafhering replied:
Done

olafhering committed:
[cmake] remove usage of kodi-platform

The pvr.pctv binary addon does not use kodi-platform.
Remove existing references from CMakeLists.txt and debian/control.
Add dependency to libp8-platform-dev because libkodiplatform-dev
pulled that dependency in as a side effect. Fix that bug too by
adding an explicit dependency.

Signed-off-by: Olaf Hering <olaf@aepfle.de>
